### PR TITLE
feat: renamed roles

### PR DESCRIPTION
**Pull request by Golbolco**
renamed per https://docs.google.com/spreadsheets/d/1U2UHpH8Tu1KhH_vjUWc2WBHPiXWjb4lJgM3_FXQ5-2g/edit#gid=0

### CODE REVIEW

File: Games/types/Mafia/items/Whiskey.js [hunk 1]
```diff
@@ -26,16 +26,19 @@ module.exports = class Sedative extends Item {
...
             if (reveal && cursed)
-              this.game.queueAlert(
-                `${this.actor.name} accidentally spiked their own drink with a sedative!`
+              this.actor.queueAlert(
+                `You couldn't resist drinking all that whiskey yourself...`
               );
             else if (reveal && !cursed)
-              this.game.queueAlert(
-                `${this.actor.name} spikes ${this.target.name}'s drink with a sedative!`
+              this.actor.queueAlert(
```
Comment (pikulet) [inline Games/types/Mafia/items/Whiskey.js:33]: why does actor need alert for who they roleblock?

File: Games/types/Mafia/items/Whiskey.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-    super("Sedative");
+    super("Whiskey");
 
     this.reveal = options?.reveal;
     this.cursed = options?.cursed;
 
     this.meetings = {
-      "Spike with Sedative": {
+      "Share Whiskey": {
         actionName: "Throw",
```
Comment (pikulet) [inline Games/types/Mafia/items/Whiskey.js:13]: throw should need changing

File: Games/types/Mafia/items/Whiskey.js (hunk 1 above)
Comment (pikulet) [reply Games/types/Mafia/items/Whiskey.js:33]: This is like snowball, previously everyone got an alert but now only Actor and target gets alerts

File: Games/types/Mafia/items/Whiskey.js [hunk 3]
```diff
@@ -26,16 +26,19 @@ module.exports = class Sedative extends Item {
...
+              this.actor.queueAlert(
+                `You share your whiskey with ${this.target.name}!`
+              ),
+              this.target.queueAlert(
+                `${this.actor.name} shares their whiskey with you!`
               );
             else
-              this.game.queueAlert(
-                `Someone spikes ${this.target.name}'s drink with a sedative!`
+              this.target.queueAlert(
```
Comment (pikulet) [inline Games/types/Mafia/items/Whiskey.js:40]: actor has no alert here?

File: Games/types/Mafia/roles/Village/Pharmacist.js [hunk 4]
```diff
@@ -7,8 +7,7 @@ module.exports = class Pharmacist extends Role {
     this.cards = [
       "VillageCore",
       "WinWithVillage",
-      "GiveSedative",
-      "SedativeImmune",
```
Comment (pikulet) [inline Games/types/Mafia/roles/Village/Pharmacist.js:11]: snowman had snowball immunity, what about pharmacist and whiskey immune

File: react_main/src/pages/Game/MafiaGame.jsx [hunk 5]
```diff
@@ -144,9 +144,6 @@ export default function MafiaGame() {
       if (winners.groups.includes("Fool")) {
         game.playAudio("music/Fool");
       }
-      if (winners.groups.includes("Mistletoe")) {
```
Comment (pikulet) [inline react_main/src/pages/Game/MafiaGame.jsx:147]: do you want matchmaker music?

File: Games/types/Mafia/items/Whiskey.js (hunk 2 above)
Comment (Golbolco) [reply Games/types/Mafia/items/Whiskey.js:13]: oops yes

**Pull request merged**